hugegraph-pd/hg-pd-dist/src/assembly/static/bin/util.sh

```
@@ -273,10 +259,22 @@ function download() {
     local link_url=$2
 
     if command_available "wget"; then
+        if [ ! -d "$path" ]; then
+            mkdir -p "$path" || {
+                echo "Failed to create directory: $path"
+                exit 1
+            }
+        fi
         wget --help | grep -q '\--show-progress' && progress_opt="-q --show-progress" || progress_opt=""
         wget ${link_url} -P ${path} $progress_opt
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)